no nie idzie

zyczynski opened the pull request:
między innymi tak kombinowałem

zyczynski committed:
no nie idzie

między innymi tak kombinowałem

telemetrik commented on tablice/trudnatablica2.php:
```diff
@@ -0,0 +1,21 @@
+    echo $imie . "</br>";
+    echo $dane['zawod'] . '</br>';
+
+        foreach ($trudnaTablica['zainteresowania'] as $zainteresowanie)
```
Czy trudnaTablica ma bezpośredni dostęp do zainteresowań? W jaki sposób dostałbyś się bez foreach do zainteresować Bartka?

zyczynski committed:
może tak?

tyle, że w przeglądarce wyświetla mi "pb", zupełnie nie wiem ocb
to zakomentowane bez foreach działa

zyczynski committed:
uff
